<launch>
  
  <!-- start camera drivers -->
  <node pkg="usb_cam" type="usb_cam_node" name="overhead_camera" output="screen" >
    <param name="video_device" type="string" value="/dev/video1"/>
  	<param name="pixel_format" type="string" value="mjpeg" />
  	<param name="image_width" type="int" value="1280" />
  	<param name="image_height" type="int" value="720" />
    <param name="camera_name" type="string" value="camera"/>
  	<param name="camera_frame_id" type="string" value="overhead_cam_frame"/>
	<param name="camera_info_url" value="file://$(find overhead_mobile_tracker)/launch/overhead_tracker_camera.yaml" />
  </node>

  <!-- <!-\- start camera drivers -\-> -->
  <!-- <node pkg="libuvc_camera" type="camera_node" name="camera_node" output="screen" > -->
  <!-- 	<param name="vendor" value="0x045e" /> -->
  <!-- 	<param name="product" value="0x0772" /> -->
  <!-- 	<param name="video_mode" type="string" value="compressed" /> -->
  <!-- 	<param name="width" type="int" value="1280" /> -->
  <!-- 	<param name="height" type="int" value="720" /> -->
  <!-- 	<param name="frame_rate" value="30.0" /> -->
  <!-- 	<param name="scanning_mode" value="1" /> -->
  <!--   <param name="timestamp_method" type="string" value="start"/> -->
  <!-- 	<param name="frame_id" type="string" value="overhead_cam_frame"/> -->
  <!-- </node> -->
  
</launch>
